roles/certificates/tasks/python-Debian.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Link CA certificates to Python
+  become: true
+  ansible.builtin.file:
+    src: /etc/ssl/certs/ca-certificates.crt
+    dest: /usr/lib/ssl/cert.pem
```

Comment:
Can you link to a bug report or show an example of the issue that this is supposed to solve? There is a symlink already from `/usr/lib/ssl/certs` to `/etc/ssl/certs`, so needing this additional symlink sounds a a packaging bug.

apt-key in particular is deprecated and should not be used any longer except to delete keys.

Finally, if indeed we do need this symlink, I think we can create it in the install-Debian playbook and don't need to add this additional one.